test/demo-autoresearch-real/run-demo.ts

```
@@ -0,0 +1,529 @@
+      console.log(
+        `${C.dim}  Calling AI to find and fix bare throws...${C.reset}`,
+      );
+
+      // AbortController + timeout to prevent hung provider calls
+      const GENERATE_TIMEOUT_MS = 180_000;
+      const controller = new AbortController();
+      const abortTimer = setTimeout(
+        () => controller.abort(),
+        GENERATE_TIMEOUT_MS,
+      );
+      let result;
```

Biome (2.4.10)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)